### PR TITLE
EPMLSTRCMW-271 Fix RunService addRun method to check ProjectId before create

### PR DESCRIPTION
**Pull request by DruzhininPavel**
**Description:**
addRun method can add Run object into any project
we must fix it.

**Changes:**
RunService - added check into addRun method
RunServiceTest - fixed test
ServiceModule - fixed creating runService

### CODE REVIEW

File: services/src/main/scala/cromwell/pipeline/service/RunService.scala [hunk 1]
```diff
@@ -37,7 +36,7 @@ object RunService {
           userId = userId,
           results = runCreateRequest.results
         )
-        runRepository.addRun(newRun)
+        projectService.getUserProjectById(runCreateRequest.projectId, userId).flatMap(_ => runRepository.addRun(newRun))
```
Comment (Nikolai-Isaev) [inline services/src/main/scala/cromwell/pipeline/service/RunService.scala:39]: If I'm right `addRun` will now cause two more exceptions from `getUserProjectById`. Shouldn't exceptions be handled more informatively  in `RunController`'s `addRun` from now on?

Comment (DruzhininPavel) [reply services/src/main/scala/cromwell/pipeline/service/RunService.scala:39]: Yep. we can add more exceptions, but I think we should do this on new task.

**Review by Nikolai-Isaev — APPROVED** [review]
Lgtm

**Review by myazinn — APPROVED** [review]

**Review by gominasian — APPROVED** [review]
Nice!

**Review by Dmitrii-Shevchenko — APPROVED** [review]

**Pull request merged**